### DISCUSSION TITLE
Possibility to add audio stream

### DISCUSSION THREAD

**Discussion by rubenixnagios** [damongolding/immich-kiosk#140] [Feature request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Would be nice for my personal use-case add Ability to attach audio/music when kiosk is launched...just an idea 
good work, keep it going!!

**Comment by bm55** [comment]
Yes, this option would be nice and it will elevate kiosk to next level!

**Comment by damongolding** [comment]
I do not believe it's possible due to the restrictions of auto playing media in browsers. 

I can do a little research into it though.
